sympy/stats/rv.py

```diff
@@ -1192,7 +1192,7 @@ def quantile(expr, evaluate=True, **kwargs):
     Quantile is defined as the value at which the probability of the random
     variable is less than or equal to the given probability.
 
-    ..math::
+    .. math::
         Q(p) = inf{x \in (-\infty, \infty) such that p <= F(x)}
```

Comment:
```suggestion
        Q(p) = \max{x \in (-\infty, \infty) : p \le F(x)}
```
I think is slightly better, although maybe not completely correct.

Reply:
@czgdp1807 Sir, Could you please confirm this once?

Reply:
I checked https://en.wikipedia.org/wiki/Quantile_function and it looks correct, except that it should be \min (or \operatorname{inf}), not \max.

Reply:
`Q(p) = \min\{x \in (-\infty, \infty) : p \le F(x)\}`
or 
`Q(p) = \inf\{x \in (-\infty, \infty) : p \le F(x)\}`
So, the above line right?
I was not knowing the operator inf or min refers to the same thanks : )

Reply:
Not sure that \inf works, but if it does then it should be OK.